examples/apply_gordo.rs

```diff
@@ -0,0 +1,68 @@
+use kube::config;
+use serde_json::Value;
+use serde_yaml;
+
+use failure::_core::time::Duration;
+use gordo_controller::crd::{gordo::load_gordo_resource, model::load_model_resource};
+use kube::api::{DeleteParams, ListParams, PostParams};
+use kube::client::APIClient;
+
+#[tokio::main]
+#[test]
+async fn main() {
+    let gordo_raw = std::fs::read_to_string(format!("{}/example-gordo.yaml", env!("CARGO_MANIFEST_DIR"))).unwrap();
+    let gordo: Value = serde_yaml::from_str(&gordo_raw).unwrap();
+
+    let client = APIClient::new(config::load_kube_config().await.unwrap());
+
+    let gordo_api = load_gordo_resource(&client, "default");
+
+    // Presently, no gordos, but after submitting one should be available
+    assert_eq!(gordo_api.list(&ListParams::default()).await.unwrap().items.len(), 0);
+    assert!(gordo_api
+        .create(&PostParams::default(), serde_json::to_vec(&gordo).unwrap())
+        .await
+        .is_ok(),);
+    assert_eq!(gordo_api.list(&ListParams::default()).await.unwrap().items.len(), 1);
+
+    // Wait for running controller to update the status
+    std::thread::sleep(Duration::from_secs(20));
```

Comment:
If you have time I think it would be cool to refactor this so it runs a loop of *max* 20 seconds where it something like
```rust
let gordo = gordo_api.get("test-project-name").await.unwrap();
if gordo.spec.config.n_models()==9 or total_runtime>20s:
 break (out of loop)
thread.sleep(0.5)
```

Reply:
Look at you, writing Rust. :cupid:

Reply:
https://github.com/equinor/gordo-controller/issues/60